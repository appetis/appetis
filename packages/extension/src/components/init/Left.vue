<template>
  <div class="w-7/12 bg-wk-dark p-14 text-white">
    <img
      :src="require('@/assets/images/workly.png')"
      alt="workly logo"
      class="h-10"
    />
    <p class="mt-3">
      We are all about the team. Organize projects, meetings, and individual
      schedules to get things done in time.
    </p>
    <div class="text-center mt-14">
      <img :src="require('@/assets/images/workly_screens.png')" alt="info" />
    </div>
  </div>
</template>
<script>
export default {
  name: 'init_left',
}
</script>
