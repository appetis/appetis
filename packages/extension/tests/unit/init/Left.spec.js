import Left from '@/components/init/Left'
import { mount } from '@vue/test-utils'

describe('init/Left', () => {
  it('code modal snapshot', () => {
    const wrapper = mount(Left)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
