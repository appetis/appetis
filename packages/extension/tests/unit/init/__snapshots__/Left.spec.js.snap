// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init/Code code modal snapshot 1`] = `
<div class="w-7/12 bg-wk-dark p-14 text-white"><img src="" alt="workly logo" class="h-10">
  <p class="mt-3">
    We are all about the team. Organize projects, meetings, and individual schedules to get things done in time.
  </p>
  <div class="text-center mt-14"><img src="" alt="info"></div>
</div>
`;

exports[`init/Left code modal snapshot 1`] = `
<div class="w-7/12 bg-wk-dark p-14 text-white"><img src="" alt="workly logo" class="h-10">
  <p class="mt-3">
    We are all about the team. Organize projects, meetings, and individual schedules to get things done in time.
  </p>
  <div class="text-center mt-14"><img src="" alt="info"></div>
</div>
`;
